crate/server/src/core/implementation.rs

```diff
@@ -453,6 +454,13 @@ impl KMS {
                     ),
                 }
             }
+            CryptographicAlgorithm::RSA => {
+                let key_size = any_attributes.cryptographic_length.ok_or(4096).unwrap();
```

Comment:
Magic number
```suggestion
                let key_size = any_attributes.cryptographic_length.ok_or(RSA_4096).unwrap();
```

Reply:
and define `RSA_4096` or could be named `RSA_DEFAULT_KEY_SIZE` in `crate/utils` where you generate RSA keys.

Reply:
No default value anymore. Returns an error if no cryptographic length has been set.